sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs

```diff
@@ -58,5 +68,23 @@ public ConfigurationClientOptions(ServiceVersion version = LatestVersion)
             };
             this.ConfigureLogging();
         }
+
+        internal string GetDefaultScope(Uri uri)
+        {
+            if (string.IsNullOrEmpty(Audience?.ToString()))
+            {
+                string host = uri.GetComponents(UriComponents.Host, UriFormat.SafeUnescaped).ToLower();
```

Comment:
I'd avoid `ToLower` here in favor of `String.EndsWith(host, StringComparison.InvariantCultureIgnoreCase)`.    The invariant culture part is relevant and some of the culture tooling flags us if we're not doing culture-aware comparisons.